autorag/nodes/queryexpansion/base.py

```diff
@@ -0,0 +1,59 @@
+import functools
+from pathlib import Path
+from typing import List, Union
+
+import pandas as pd
+
+from autorag import generator_models
+
+from autorag.utils import result_to_dataframe, validate_qa_dataset
+
+import logging
+
+logger = logging.getLogger("AutoRAG")
+
+
+def query_expansion_node(func):
+    @functools.wraps(func)
+    @result_to_dataframe(["expanded_queries"])
```

Comment:
The column name must be 'queries'.
It will occur error at retrieval module.
You must change this to 'queries', or you can change things at retrieval node decorator to 'expanded_quries'